KuFlow.UiPathLibrary/KuFlow.UiPathLibrary.Activities/Properties/Resources.Designer.cs

```diff
@@ -8,141 +8,857 @@
+        /// </summary>
+        public static string SaveElementDocument_Description {
+            get {
+                return ResourceManager.GetString("SaveElementDocument_Description", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Save element document.
+        /// </summary>
+        public static string SaveElementDocument_DisplayName {
+            get {
+                return ResourceManager.GetString("SaveElementDocument_DisplayName", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to A file path to upload in a file system.
+        /// </summary>
+        public static string SaveElementDocument_FilePath_Description {
+            get {
+                return ResourceManager.GetString("SaveElementDocument_FilePath_Description", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to File Path.
+        /// </summary>
+        public static string SaveElementDocument_FilePath_DisplayName {
+            get {
+                return ResourceManager.GetString("SaveElementDocument_FilePath_DisplayName", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to UUID to identify a document. Leave blanck to generate one..
+        /// </summary>
+        public static string SaveElementDocument_Id_Description {
+            get {
+                return ResourceManager.GetString("SaveElementDocument_Id_Description", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Id.
+        /// </summary>
+        public static string SaveElementDocument_Id_DisplayName {
+            get {
+                return ResourceManager.GetString("SaveElementDocument_Id_DisplayName", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Task resource.
+        /// </summary>
+        public static string SaveElementDocument_Task_Description {
+            get {
+                return ResourceManager.GetString("SaveElementDocument_Task_Description", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Task.
+        /// </summary>
+        public static string SaveElementDocument_Task_DisplayName {
+            get {
+                return ResourceManager.GetString("SaveElementDocument_Task_DisplayName", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Identifier of the task.
+        /// </summary>
+        public static string SaveElementDocument_TaskIdentifier_Description {
+            get {
```

Comment:
Is this file well formatted? in other files the `{` goes in a new line

Reply:
Yes you're right. However this is an autogenerated file. That is, I put the keys and values as in a *.properties file in JAVA, and then the IDE transforms it to the C# file. I never edit this final file.